util/printchplenv

```diff
@@ -132,6 +128,32 @@ def print_mode(mode='list'):
         print_mode('runtime')
         stdout.write('CHPL_MAKE_LAUNCHER_SUBDIR=')
         print_mode('launcher')
+
+        third_party_pkgs = chpl_home+'/util/chplenv/third-party-pkgs'
```

Comment:
This doesn't fall on you, but we should really refactor third-party-pkgs and rename it so it can just be imported. :-\